mkosi/__init__.py

```diff
@@ -1916,6 +1972,9 @@ def install_kernel(context: Context, partitions: Sequence[Partition]) -> None:
         if context.config.bootloader == Bootloader.uki:
             break
 
+    if want_uki(context):
+        install_uki_addons(context)
```

Comment:
I think we can do this earlier after `run_hwdb()` or so. I don't think we should gate this on `want_uki()`. If the addons are configured, let's generate them. That allows users to disable `Bootable=` but still generate addons in case they want to generate UKIs via their own logic